Formula/flix.rb

```diff
@@ -6,6 +6,11 @@ class Flix < Formula
   license "Apache-2.0"
   head "https://github.com/flix/flix.git", branch: "master"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^v?\.?(\d+(?:\.\d+)+)$/i)
```

There are a couple of older tags where they used a `v.` prefix instead of `v` (e.g., `v.0.8.1`, `v.0.9.0`, instead of the usual `v0.28.0`), so it would probably be good to account for that possibility in this particular context.

Reply:
👍